src/cli_arguments.rs

```diff
@@ -539,3 +583,286 @@ impl CliArguments {
         }
     }
 }
+
+#[derive(Default)]
+struct EnvDefaults {
```

Comment:
I wonder - maybe instead of this struct it'd be better to mutate the `EvalutatedCall` object?